gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-core/src/main/java/io/gravitee/am/gateway/handler/root/resources/handler/login/LoginCallbackOpenIDConnectFlowHandler.java

```diff
@@ -69,6 +70,12 @@ public void handle(RoutingContext context) {
                 context.next();
                 return;
             }
+            // if OAuth 2.0 authorization_code flow is used, continue
+            final String code = request.getParam(Parameters.CODE);
```

Comment:
Is this to prevent the callback from happening if we are using authorization code ?

Reply:
Yes indeed